sql/mysql/driver.go

```diff
@@ -90,6 +88,13 @@ func (d *Driver) NormalizeSchema(ctx context.Context, s *schema.Schema) (*schema
 	return d.dev().NormalizeSchema(ctx, s)
 }
 
+// ChangeSchema alters the given URL, so it connects to the given schema name.
+func (d *Driver) ChangeSchema(u *sqlclient.URL, s string) *sqlclient.URL {
```

Comment:
May I suggest putting this logic on the sqlclient API and not the driver. It does not feel right to expose this functionality on am opened Driver.